lib/services/api/api_service.ts

```diff
@@ -188,7 +188,9 @@ export class SNApiService extends PureService {
       url,
       undefined,
       this.session!.accessToken
-    );
+    ).then(async (response) => {
```

Comment:
This is redundant, as the response will be what is returned by the return statement on 187. This would only be useful if you want to modify response before returning it or if you're wanting to return a subset of the response object.

Reply:
My bad, I was debugging the response object and forgot to remove this :sweat_smile: